src/wp-includes/blocks.php

```diff
@@ -753,23 +759,24 @@ function excerpt_remove_blocks( $content ) {
 }
 
 /**
- * Render inner blocks from the `core/columns` block for generating an excerpt.
+ * Render inner blocks from the allowed wrapper blocks
+ * for generating an excerpt.
  *
- * @since 5.2.0
+ * @since 5.8
  * @access private
  *
- * @param array $columns        The parsed columns block.
+ * @param array $parsed_block   The parsed block.
  * @param array $allowed_blocks The list of allowed inner blocks.
  * @return string The rendered inner blocks.
  */
-function _excerpt_render_inner_columns_blocks( $columns, $allowed_blocks ) {
+function _excerpt_render_inner_blocks( $parsed_block, $allowed_blocks ) {
 	$output = '';
 
-	foreach ( $columns['innerBlocks'] as $column ) {
-		foreach ( $column['innerBlocks'] as $inner_block ) {
-			if ( in_array( $inner_block['blockName'], $allowed_blocks, true ) && empty( $inner_block['innerBlocks'] ) ) {
-				$output .= render_block( $inner_block );
-			}
+	foreach ( $parsed_block['innerBlocks'] as $inner_block ) {
+		if ( in_array( $inner_block['blockName'], $allowed_blocks, true ) ) {
```

Comment:
The ternary here is pretty hard to read. May I suggest using `continue` to bail early, and then an `if` statement. Something like:

```
if ( ! in_array( $inner_block['blockName'], $allowed_blocks, true ) ) {
	continue;
}

if ( empty( $inner_block['innerBlocks'] ) {
	render_block( $inner_block );
} else {
	_excerpt_render_inner_blocks( $inner_block, $allowed_blocks );
}
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/1472/commits/c184975132994210e36e12a6f096e7e728f9f88a :+1: